public/assets/data/patientEducationReferences.json

```diff
@@ -41,7 +41,7 @@
                 "link": {
                     "title": "Naloxone Patient / Public",
                     "className": "education",
-                    "url": "https://www.doh.wa.gov/Portals/1/Documents/2300/2019/630146.pdf",
+                    "url": "https://doh.wa.gov/sites/default/files/legacy/Documents/2300/2019//630146.pdf?uid=624f125ed218b",
```

Comment:
I see an extra forward slash just before the pdf filename?

Reply:
thanks @pbugni.  fixed.